lib/valkyrie/storage/fedora.rb

```diff
@@ -79,8 +81,13 @@ def fedora_identifier(id:)
       RDF::URI(identifier)
     end
 
-    def id_to_uri(id)
-      RDF::URI("#{connection_prefix}/#{CGI.escape(id.to_s)}")
+    # Convert the simple id into a URI
+    # @param [String] simple id (e.g. 'AN1D4UHA')
+    # @param id_transformer [lambda] procedure for performing the transformation
+    # @return [RDF::URI] the transformed URI (e.g. <RDF::URI 'http://localhost:8998/rest/dev/AN1D4UHA'>)
+    def id_to_uri(id, id_transformer: default_id_transformer)
```

Comment:
I wonder if the id_transformer should just take a resource, so we get rid of this method and up top have `id_transformer.call(resource: resource)`, which should return an `RDF::URI`. Maybe then it's a ResourceURITransformer?

Reply:
done